### ISSUE TITLE
List of events shouldn't show events on default

### ISSUE THREAD

**Issue by kpenning** [aclu-people-power/map#99]
List of events shouldn't show any events until the user has inputted a zip code or zoomed to an area. See functionality at current map for example. Right now I get these events pre-loaded despite the fact that I live somewhere else:

![image](https://cloud.githubusercontent.com/assets/4789544/25454675/50a961f0-2a9b-11e7-9fbe-d03a4d2a387d.png)

**Comment by stuartsan** [comment]
@kpenning , yeah so right now the list is just showing all the events, 1:1 correspondence with what is shown on the map.

The way this design works there is always a white sidebarlike thing on the right, even when no events show up. So I'm wondering -- what should we put there? Maybe a count of the total number of events, total events created today, something like that? I think totally empty space will look not great.

Should we consider this critical for V1 or ok to ship afterwards?

cc also @mollieru
